include/hipSYCL/sycl/libkernel/sscp/builtins.hpp

```diff
@@ -176,7 +176,7 @@ HIPSYCL_BUILTIN float __hipsycl_lgamma_r(float x, IntT* ptr) {
 }
 
 template<class IntT>
-HIPSYCL_BUILTIN double __hipsycl_lgamma_r_f64(double x, IntT* ptr) {
+HIPSYCL_BUILTIN double __hipsycl_lgamma_r(double x, IntT* ptr) {
```

Comment:
Wouldn't we also need a float overload that invokes `__hipsycl_sscp_lgamma_r_f32()`?

Reply:
Ah, I got it. Sorry! :)